README.md

```diff
@@ -1,6 +1,8 @@
 # phpList core module
 
 [![Build Status](https://travis-ci.org/phpList/core.svg?branch=master)](https://travis-ci.org/phpList/core)
+[![Code Coverage](https://scrutinizer-ci.com/g/phpList/core/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/phpList/core/?branch=master)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/phpList/core/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/phpList/core/?branch=master)
```

Comment:
Please use the badges from https://shields.io/ instead - they're also available as SVG (which the Scrutinizer-provided badges aren't, making the badges look blurry on high-resolution screens).

Reply:
I wish you tell that earlier ;-)

Reply:
I had, had I known that Scrutinizer itself does only provide PNG badges (which I found out by reading this PR and then checking for SVG badges). So, barring a time machine, I'm sorry I couldn't tell you this earlier. ;-)